hyp3_gamma/insar/ifm_sentinel.py

```diff
@@ -265,7 +270,8 @@ def make_parameter_file(mydir, parameter_file_name, alooks, rlooks, dem_source):
 
 
 def insar_sentinel_gamma(reference_file, secondary_file, rlooks=20, alooks=4, include_look_vectors=False,
-                         include_los_displacement=False, include_wrapped_phase=False, include_inc_map=False):
+                         include_los_displacement=False, include_wrapped_phase=False, include_inc_map=False,
+                         include_dem=False):
```

Comment:
changelog calls out "--include-dem` parameter to `insar` entrypoint", should probably do that as part of this PR